packages/cli/src/config/extension.test.ts

```diff
@@ -454,6 +450,92 @@ describe('extension tests', () => {
       expect(consoleSpy).toHaveBeenCalledWith('Extension not found: ext4');
       consoleSpy.mockRestore();
     });
+
+    describe('autoUpdate', () => {
+      let tempHomeDir: string;
```

Comment:
you should be able to use the top-level one and remove this and the autoUpdate beforeEach + afterEach